docs/source/build_network.rst

```diff
@@ -1,12 +1,10 @@
 Building Your First Network
 ===========================
 
-.. note:: These instructions have been verified to work against the
-          latest stable Docker images and the pre-compiled
-          setup utilities within the supplied tar file. If you run
-          these commands with images or tools from the current master
-          branch, it is possible that you will see configuration and panic
-          errors.
+.. note:: The Build your first network (BYFN) tutorial is being deprecated. If
```

Comment:
I would remove the word "being", a deprecation is a notice of future removal, so it implies "being".

Reply:
changed to `has been deprecated`